packages/ui/Scrollable/Scrollable.js

```diff
@@ -791,13 +794,17 @@ class ScrollableBase extends Component {
 		const
 			bounds = this.getScrollBounds(),
 			maxValue = bounds.maxLeft,
-			{type, ratio} = this.getOverscrollStatus('horizontal');
+			orientation = 'horizontal';
 
 		this.scrollLeft = clamp(0, maxValue, value);
 
-		if (type === overscrollTypes.scrolling) {
-			this.updateOverscrollEffect('horizontal', this.scrollLeft, type, ratio);
-		}
+		['before', 'after'].forEach((position) => {
+			const {type, ratio} = this.getOverscrollStatus(orientation, position);
+
+			if (type === overscrollTypes.scrolling) {
```

Comment:
I don't think we need this condition because we already did with the following condition in `updateOverscrollEffectByDirection()`.
```
if (ratio > 0 && type === scrolling)
```

Reply:
Yes, we need this due to cases through `updateOverscrollEffect()`.

Reply:
If we could omit the condition here, we could just call this.updateOverscrollEffectByDirection(orientation, position). Then type and ratio could be get from this.getOverscrollStatus(orientation, position) not out of but in this.updateOverscrollEffectByDirection().

Reply:
My intention is not to call another function since this routine is possibly called frequently especially on dragging.